app/src/main/java/com/example/seonoh/seonohapp/CoinAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.seonoh.seonohapp
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.seonoh.seonohapp.model.CurrentPriceInfoModel
+
+class CoinAdapter(data: ArrayList<CurrentPriceInfoModel>,val type : Int) : RecyclerView.Adapter<KrwItemViewHolder>(){
+
+    lateinit var mView : View
+    val mData = data
```

Comment:
```suggestion
class CoinAdapter(private val data: ArrayList<CurrentPriceInfoModel>,val type : Int) : RecyclerView.Adapter<KrwItemViewHolder>(){
```

- mData변수의 접근자를 private으로 변경하는 것이 좋을 것 같습니다.
- data를 파라미터로 전달 받아, 클래스 변수에 넣는 것보단 주 생성자에서 클래스 변수로 두는 것이 좋을 것같습니다.

Reply:
감사합니다 ! 리뷰를 보니 제가 평소에 private을 자주 사용하지 않는 것을 느꼈습니다.